ethereum/spec/src/main/java/tech/pegasys/teku/spec/util/CommitteeUtil.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.util;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static tech.pegasys.teku.datastructures.util.BeaconStateUtil.bytes_to_int64;
+import static tech.pegasys.teku.datastructures.util.BeaconStateUtil.uint_to_bytes;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.crypto.Hash;
+import tech.pegasys.teku.spec.constants.SpecConstants;
+
+public class CommitteeUtil {
+  private final SpecConstants specConstants;
+
+  public CommitteeUtil(final SpecConstants specConstants) {
+    this.specConstants = specConstants;
+  }
+
+  /**
+   * Return the shuffled validator index corresponding to ``seed`` (and ``index_count``).
+   *
+   * @param index
+   * @param index_count
+   * @param seed
+   * @return
+   * @see
+   *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.0/specs/core/0_beacon-chain.md#is_valid_merkle_branch</a>
```

Comment:
This link is a long way out of date.  Let's just drop the links to the spec as we migrate things.

Reply:
ok sounds fair i had considered just removing the doc portions?

Reply:
I'd be happy with that too.